docs/concepts/endpoint-structure/request-response-structure.md

```diff
@@ -12,11 +12,68 @@ From this base endpoint, we can assemble almost all paths described in the subse
 
 ## Request format
 
-In Shopware 6, the request body has to be JSON encoded. Always set the `Content-Type` header to `application/json` if you're sending data to the API.
+The request format for Shopware 6 store API's must follow the below format:
 
-It's mandatory to use type-safe values, e.g. if the API expects an integer value, you're required to provide an actual integer. If you're using a Date field, make sure to use an ISO 8601 compatible date format.
+* The request body has to be JSON encoded.
 
-```javascript
+* It is mandatory to use type-safe values, e.g., if the API expects an integer value, you are required to provide an actual integer.
+
+* When using a Date field, make sure to use an ISO 8601 compatible date format.
+
+### Request URL
+
+The Admin API can be separated into two general sets of endpoints
+
+* Generic entity endpoints
+
+* Specific interaction/configuration endpoints
+
+Both fulfill specific cases which are outlined below.
+
+#### Generic entity endpoints
+
+Provide CRUD functionalities on **all entities available** in Shopware. Go to the [Entity Reference](../resources/entity-reference.md) section to see a list of all available entities and their structure.
+
+Furthermore, these endpoints can be divided into **read** and **write** operations. To learn more about those, please head to the corresponding guide on [Reading Entities](../guides/reading-entities.md) and [Writing Entities](../guides/writing-entities/README.md). These endpoints are generic, since they are entirely based on the entities definitions and contain no additional business logic besides data validation.
+
+> The URL structure for *Generic entity endpoints* is:
+> ```
+> /api/{entity-name}(/{entity-id})
+> ```
+
+#### Specific interaction or configuration endpoints
+
+Provide interactions for more sophisticated operations which can change the state of the system but are not directly based on entities alone. We differ between more than 20 categories of about 140 specific endpoints, such as
+
+* Document Management
+* Order State Management
+* User Role Management
+* System Operations
+* Authorization & Authentication
+
+which will let you automate every part of the store operation.
+
+> The URL structure for *Specific interaction or configuration endpoints* roughly follows this schema:
+> ```
+> /api/_action/{interaction-name}
+> ```
+
+### Request headers
+
+Request headers provide information about your request to a REST API service that allows you to authenticate/authorize and receive the request body.
```

Comment:
I would remove this from the ADMIN api since the authorization does not happen via headers, but the auth endpoint with a USER or Integration and from there with a bearer token